### ISSUE TITLE
Feature request: ChoiceBlock should allow multiple choices

### ISSUE THREAD

**Issue by kunambi** [wagtail/wagtail#3572]
It could be another parameter e.g. `multiple=True`.

### Example
```
    align = blocks.ChoiceBlock(label=_('Align'), required=True, choices=[
        ('align-left', _('Left')),
        ('align-right', _('Right')),
    ], help_text=_('On which side should the image show?'), multiple=True)
```

### Expected behaviour:
Print the choices as a series of `<input type="checkbox">` in the Wagtail admin

### Expected result:
Send the chosen values as an iterable list to the template

**Comment by gasman** [comment]
Thanks for the suggestion @kunambi! I think a separate `MultipleChoiceBlock` block type would be a better fit for this, since the process for outputting multiple values on the template (and storing them in the database) would be substantially different from handling a single value.

**Comment by gasman** [comment]
Implemented in #5592 / #5819
